src/trees/MWTree.h

```diff
@@ -103,6 +103,9 @@ template <int D> class MWTree {
     void makeNodeTable(MWNodeVector<D> &nodeTable);
     void makeNodeTable(std::vector<MWNodeVector<D>> &nodeTable);
 
+    int getNodeTableSize() const { return this->endNodeTable.size(); }
+    int getNNodesAtDepth(int i) const { return this->nodesAtDepth[i]; }
```

Comment:
and `getNNodes(int depth)`...

Reply:
oh god yes again! I'll kill this one and update the vampyr counterpart